src/__swaps__/screens/Swap/hooks/useSearchCurrencyLists.ts

```diff
@@ -27,6 +27,14 @@ export interface AssetToBuySection {
 const MAX_UNVERIFIED_RESULTS = 8;
 const MAX_VERIFIED_RESULTS = 48;
 
+const mergeAssetsFavoriteStatus = ({
+  assets,
+  favoritesList,
+}: {
+  assets: SearchAsset[] | undefined;
+  favoritesList: SearchAsset[] | undefined;
+}): SearchAsset[] => assets?.map(asset => ({ ...asset, favorite: favoritesList?.some(fav => fav.address === asset.address) })) || [];
```

Comment:
we were seeing bridge / recents sections not showing proper favorites status because we weren't adding the favorite boolean